manim_slides/present.py

```diff
@@ -90,23 +90,41 @@ def __init__(self, config: PresentationConfig) -> None:
     def __len__(self) -> int:
         return len(self.slides)
 
+    @property
+    def slides(self) -> List[SlideConfig]:
+        """Returns the list of slides."""
+        return self.config.slides
+
+    @property
+    def files(self) -> List[Path]:
+        """Returns the list of animation files."""
+        return self.config.files
+
+    @property
+    def resolution(self) -> Tuple[int, int]:
+        """Returns the resolution."""
+        return self.config.resolution
+
```

Comment:
@Fairlight8 Second, add a property to `Presentation` so that `Presentation.resolution` maps to `PresentationConfig.resolution`